index.js

```diff
@@ -16,14 +19,21 @@ const appPort = process.env.PORT || 3001
 
 const bodyParser = require('body-parser')
 
-msLogger.setLogLevel(process.env.LOG_LEVEL || msLogger.LOG_LEVELS.DEBUG)
+logger.setLogLevel(process.env.LOG_LEVEL || logger.LOG_LEVELS.DEBUG)
 
 const app = express()
 app.use(bodyParser.json())
 app.use(msCorrelation.correlationMiddleware)
+app.use(msLogger.middleware)
+
+if (process.env.DEV) {
+  app.use(authenticationDevMiddleware())
```

Comment:
You're using the authentication on any request. This means that any request with a user that is not authenticated will be redirected, not just the UI.
I now notice that the UI doesn't exist yet in this PR. You can just put here a TODO until it is. Also, is all CRUD operations that the UI allows are only available to admins, maybe the entire UI should only be available to admins...

Reply:
Ok ,first of all, Who is responsible for the login ui?
Second, Thats why you saw the cors thing, It wont  be here after the ui would be served from the node server

Reply:
I'm responsible for the login UI.
I didn't get the second point. Could you explain?

Reply:
The cors thing was in the other pr's because of the client.
If the node server is serving the client the cors is not needed any more.

Reply:
You do need the cors because you need other services to send requests to your API, and different ports require CORS